apps/mobile-wallet/src/features/modals/BottomModal2.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  BottomSheetBackdropProps,
+  BottomSheetModal,
+  BottomSheetModalProps,
+  BottomSheetView
+} from '@gorhom/bottom-sheet'
+import { useCallback, useEffect, useRef } from 'react'
+import { useTheme } from 'styled-components/native'
+
+import BottomModalBackdrop from '~/features/modals/BottomModalBackdrop'
+import { BottomModalBaseProps } from '~/features/modals/BottomModalBase'
+import BottomModalHandle from '~/features/modals/BottomModalHandle'
+import BottomModalHeader from '~/features/modals/BottomModalHeader'
+import { closeModal } from '~/features/modals/modalActions'
+import { useAppDispatch } from '~/hooks/redux'
+import { DEFAULT_MARGIN, VERTICAL_GAP } from '~/style/globalStyle'
+
+export type BottomModal2Props = BottomSheetModalProps & BottomModalBaseProps
```

Comment:
I am using the same props as the ones we had before so that all our modals can be migrated as easily as possible